tests/acceptance/let-helper-test.js

```diff
@@ -59,4 +61,21 @@ describe('Acceptance: let helper', function() {
       });
     });
   });
+
+  if (emberVersionIs('greaterThan', "2.0.0")) {
```

Comment:
Yes, it's in devDependencies and only included in the test

--
Taras Mankovski
@tarasm




On Thu, Aug 25, 2016 at 10:12 PM +0530, "Charles Lowell" <notifications@github.com> wrote:












In tests/acceptance/let-helper-test.js:
> @@ -59,4 +61,21 @@ describe('Acceptance: let helper', function() {
>        });
>      });
>    });
> +
> +  if (emberVersionIs('greaterThan', "2.0.0")) {



It's only brought in at test time, right?



—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or mute the thread.